### PR TITLE
Create post for October Meetup

### PR DESCRIPTION
**Pull request by mariostallone**

### CODE REVIEW

File: _posts/2020-10-01-october-meetup.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "October 2020"
+date:   2020-10-01 00:00:00
```
Comment (JoachimK) [inline _posts/2020-10-01-october-meetup.md]: The date here and the date in the filename have to be in the past for Jekyll to pick it up. Sorry, it's confusing, it makes sense to put the date of the actual CocoaHeads, but that's not how the blog engine of GitHub Pages works :/

Comment (mariostallone) [reply _posts/2020-10-01-october-meetup.md]: Gotcha. Thanks for the quick review. Fixing!

File: _posts/2020-10-01-october-meetup.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+19:30 - Meeting starts 🎉
+
+19:45 - 1️⃣
+Alexander: Silicon Soul: The Machine we must not talk about
+
+Apple introduced the Mac’s transition to ARM64, and thus the fourth CPU architecture after Motorola 68k, PowerPC and Intel.
+Apart from the question what this means for the platform this is a welcome opportunity to look deep beyond the surface of technologies like SwiftUI.
+Deep below we want to explore how the dynamic loader dyld brings apps to life from files containing machine code. This includes a primer into ARM64 assembler, the language now used on almost all Apple devices.
+Finally there is the question: “Hour does knowing all of this help me in my daily work?”
```
Comment (JoachimK) [inline _posts/2020-10-01-october-meetup.md]: Typo (by Alex)
```suggestion
Finally, there is the question: “How does knowing all of this help me in my daily work?”
```

File: _posts/2020-09-27-october-meetup.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Feel free to join us via Slack, where we'll keep you up-to-date on any immediate announcements.
+👉🏼 [Click here](https://slack.cocoaheads.hamburg) to join our workspace 'cocoaheadshamburg.slack.com'
+
+19:20 - You can start joining the Video Call
+
+19:30 - Meeting starts 🎉
+
+19:45 - 1️⃣
+Alexander: Silicon Soul: The Machine we must not talk about
```
Comment (JoachimK) [inline _posts/2020-09-27-october-meetup.md]:
```suggestion
Alexander von Below: Silicon Soul: The Machine we must not talk about
```
Let's put his full name here, as I think many people will only recognize him by his last name. Once we published it, we can double-check with Alex whether he wants us to put it somehow differently.

File: _posts/2020-10-01-october-meetup.md (hunk 2 above)
Comment (mariostallone) [reply _posts/2020-10-01-october-meetup.md]: Thanks

File: _posts/2020-09-27-october-meetup.md (hunk 3 above)
Comment (mariostallone) [reply _posts/2020-09-27-october-meetup.md]: Sure. I wasn't sure of this being his Full Name, thanks for confirming!

**Review by JoachimK — APPROVED** [review]

**Pull request merged**